packages/loot-core/src/server/importers/ynab4.ts

```diff
@@ -424,12 +400,9 @@ export async function importBuffer(filepath, buffer) {
     throw new Error('Error reading Budget.yfull file');
   }
 
-  let data;
   try {
-    data = JSON.parse(contents);
+    return JSON.parse(contents);
   } catch (e) {
-    throw new Error('Error parsing Budget.yull file');
+    throw new Error('Error parsing Budget.yfull file');
```

Comment:
Was this change intentional?

Reply:
Yep, the file extension is `yfull` full so this must have been a typo.